doc/classes/OS.xml

```diff
@@ -1162,6 +1172,34 @@
 		<constant name="MONTH_DECEMBER" value="12" enum="Month">
 			December.
 		</constant>
+		<constant name="APPLICATION_HANDLE" value="0" enum="HandleType">
+			Application handle: 
```

Comment:
You have a trailing space on each first line of the constant descriptions.

Reply:
yeah I noticed, already fixed :)